oci/pack.go

```diff
@@ -163,9 +163,30 @@ func NewPackageFromTarget(ctx context.Context, targ target.Target, opts ...packm
 
 	if popts.Initrd() != "" {
 		log.G(ctx).Debug("oci: including initrd")
+		initRdPath := popts.Initrd()
+		if f, err := os.Stat(initRdPath); err == nil && f.IsDir() {
+			cwd, err2 := os.Getwd()
+			if err2 != nil {
+				return nil, err2
+			}
+
+			file, err := os.CreateTemp("", "kraftkit-oci-archive-*")
```

Comment:
I just noticed we somehow have to clean this file up once it has been packaged.

Maybe introduced a higher-level string slice, `cleanUp`, that on `defer` is iterated on and each file is removed in this slice.

Reply:
We never clean up after ourselves, when I package stuff tmp it's always filled with junk :D

Reply:
I'll open an issue that tackles looking for and handling cleanups. In the mean time, we can solve the first here :)

Reply:
Rendered in #679

Reply:
check again